### ISSUE TITLE
Investigate search tuning and UX improvements

### ISSUE THREAD

**Issue by adborden** [GSA/data.gov#598]
Spend some time looking at the UX experience around search, what can be done in the UI and for tuning the backend.

Some issues that we've seen:
- For datasets with a date in the title, users can’t sort by date.
- Some datasets are missing tags that help hint they are authoritative.
- Should we de-prioritize city/county over federal datasets?
- What solr search tuning can we do to yield better results?
- How can we better highlight solr's query syntax so they can create better searches? E.g. field search `title:(federal lands)`

This came up in our [retro discussion around search relevance](https://docs.google.com/document/d/16tJ3f-NgZJcAVtmkvK8bI45JDMhwQKJG9RaR4HD-28U/edit#heading=h.gwfq8eet78qq).

**Comment by jbrown-xentity** [comment]
Tagging @hkdctol and @CarolinaC-REI for relevance in new UI discussion. Can probably close after wrapping into whatever design/analysis you are currently doing.

**Comment by hkdctol** [comment]
agree--maybe link this to whatever ticket is appropriate now or in future
